formats/cbor/commonTest/src/kotlinx/serialization/cbor/CborReaderTest.kt

```diff
@@ -13,9 +15,9 @@ class CborReaderTest {
 
     private val ignoreUnknownKeys = Cbor { ignoreUnknownKeys = true }
 
-    private fun withDecoder(input: String, block: CborDecoder.() -> Unit) {
+    private fun withDecoder(input: String, block: CborParser.() -> Unit) {
```

Comment:
minor, optional: class is now called `CborParser` so both test name and function can be renamed

Reply:
Split up into tow distinct Test classes in 46227cf03e0e5af9e484624b4036d9eb75a542d4